draft-ietf-tls-tls13.md

```diff
@@ -4695,22 +4695,33 @@ concerned with:
   handshake with B and probably retry the request, leading to duplication on
   the server system as a whole.
 
-The first class of attack can be prevented by the mechanism described
-in this section.  Servers need not permit 0-RTT at all, but those
-which do SHOULD implement either the single-use tickets or
-ClientHello recording techniques described in the following two
-sections.
+The first class of attack can be prevented by sharing state in orter to provide
+a guarantee that the 0-RTT is only accepted once.  Servers SHOULD provide that
```

Comment:
I don't want to write "0-RTT data", because this could imply, for instance, that a server could implement a strike register keyed on the 0-RTT payload.